add-results-task-pane

dani-ras opened the pull request:
I added a custom task pane for reviewing the search and replace results and optionally undo each one.
It shows after a call to `CensorNames` was done,
each result is highlighted in the doc,
clicking on a result will select the replacement text in the doc.

![image](https://github.com/Pituchey-Hotam/Genizah/assets/77384873/c231a794-bd7f-4987-bd20-8a4556b2bc5e)

Commit:
add-results-task-pane

michael-3-141 commented:
עבודה מצוינת!
מה שכדאי לתקן לפני שאמרג'ג':
- [ ] לתקן את הבאג שרק מופע אחד מכל שם נמצא בכל לחיצה
- [ ] להסיר את קוד החיפוש הישן, ושגם חיפוש לפני הדפסה ישתמש בtask pane. לא רואה טעם לשמור את שניהם.

dani-ras committed:
modify FindReplace and improve design
FindReplace:
catch all shows
single replace in 1 word
design:
add tooltips
adjust widths

dani-ras committed:
modify FindReplace and improve design
FindReplace:
catch all shows
single replace in 1 word
design:
add tooltips
adjust widths
